target/common/common.mk

```diff
@@ -86,7 +87,13 @@ VLT_FLAGS    += -Wno-UNSIGNED
 VLT_FLAGS    += -Wno-UNOPTFLAT
 VLT_FLAGS    += -Wno-fatal
 VLT_FLAGS    += --unroll-count 1024
+ifneq ($(VLT_NUM_THREADS), 1)
+	VLT_FLAGS	 += --threads $(VLT_NUM_THREADS)
+endif
```

Comment:
I believe we can drop the `if` statement, as [Verilator should by default behave as if `--threads 1` is passed](https://github.com/pulp-platform/snitch_cluster/assets/17752123/7863d1e1-479b-4dfb-a83a-97c2bb6e5b28).

Reply:
I believe this is true for Verilator 5 but not the verilator 4 that the repo currently supports until https://github.com/pulp-platform/snitch_cluster/pull/76 lands.
That said I think you're right and we can drop this as this is almost certainly a premature optimization and I doubt single-threaded usage will see a measureable difference.